PixaPicker/ViewController.swift

```diff
@@ -11,15 +11,12 @@
 import UIKit
 import SDWebImage
 
-class ViewController: UIViewController, UISearchResultsUpdating, UISearchBarDelegate, UICollectionViewDataSource, UICollectionViewDelegate, UICollectionViewDelegateFlowLayout{
+class ViewController: UIViewController, UISearchResultsUpdating, UISearchBarDelegate, UICollectionViewDataSource, UICollectionViewDelegate, UICollectionViewDelegateFlowLayout, PixaDataCoordinatorDelegate{
     
     @IBOutlet weak var imageCollectionView: UICollectionView!
     
     let reuseID = "cell"
-    var cellImageURLs = [URL]()
-    var currentPageNumber = 1 //dataCoordinator
-    var currentSearchText = ""
-    let maxItemsPerPage = 20
+    var dataCoordinator:PixaDataCoordinator?
```

Comment:
I would instead have it as `let dataCoordinator = PixaDataCoordinator()` unless I'm missing something